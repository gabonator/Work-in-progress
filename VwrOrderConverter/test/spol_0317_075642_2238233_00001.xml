<?xml version="1.0" encoding="UTF-8"?><Envelope xmlns="http://schemas.microsoft.com/dynamics/2008/01/documents/Message"><Header><MessageId>{CE0E643B-2B37-47FF-BE04-711DCCBF0942}</MessageId><SourceEndpoint>Spolek</SourceEndpoint><DestinationEndpoint>AIFPurch_D71956</DestinationEndpoint><Action>PURCHORDER</Action></Header><Body><MessageParts xmlns="http://schemas.microsoft.com/dynamics/2008/01/documents/Message"><PurchaseRequisition xmlns="http://schemas.microsoft.com/dynamics/2008/01/documents/PurchaseRequisition"><DocPurpose>Original</DocPurpose><SenderId>spol</SenderId><VendPurchOrderJour class="entity"><_DocumentHash>23e0544dbec9a689a914d9931b342ec9</_DocumentHash><AddressCode>00011789</AddressCode><Amount>2221.56</Amount><AmountPurchaseOrder>Yes</AmountPurchaseOrder><CountryRegionId>CZ</CountryRegionId><CurrencyCode>CZK</CurrencyCode><DeliveryAddress>Kekulova/Sklářská
400 32  Ústí nad Labem</DeliveryAddress><DeliveryCity>Ústí nad Labem</DeliveryCity><DeliveryCountryRegionId>CZ</DeliveryCountryRegionId><DeliveryDate>2020-03-31</DeliveryDate><DeliveryName>Spolek pro chemickou a hutní výrobu, akciová společnost</DeliveryName><DeliveryState>42</DeliveryState><DeliveryStreet>Kekulova/Sklářská</DeliveryStreet><DeliveryType>None</DeliveryType><DeliveryZipCode>400 32</DeliveryZipCode><Dimension><element /><element>10902</element><element /><element>SPO</element></Dimension><ExchRate>100.000000000000</ExchRate><InclTax>No</InclTax><InterCompanyPosted>No</InterCompanyPosted><InvoiceAccount>D71956</InvoiceAccount><ItemBuyerGroupId>U01</ItemBuyerGroupId><LanguageId>CS</LanguageId><OrderAccount>D71956</OrderAccount><ParmId>9173009</ParmId><PurchaseOrderId>SP20-03595</PurchaseOrderId><PurchId>N20-03974</PurchId><PurchOrderDate>2020-03-17</PurchOrderDate><PurchOrderDocNum>N20-03974-1</PurchOrderDocNum><Qty>4.000</Qty><RecId>5638284652</RecId><RecVersion>376390203</RecVersion><Requisitioner>03336</Requisitioner><SalesOrderbalance>1836.00</SalesOrderbalance><Status>Created</Status><SumTax>385.56</SumTax><Triangulation>No</Triangulation><VendGroup>TUZ</VendGroup><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2020-03-31</DeliveryDate><Dimension><element /><element>10902</element><element /><element>SPO</element></Dimension><EmplEDIEmail>kilianova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Kilianová Jana</EmplEDIEmplName><EmplEDIOffice>10902</EmplEDIOffice><EmplEDIPhone>+420477163709</EmplEDIPhone><ExternalItemId>24575.320</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>4.00</InventQty><InventTransId>ID03993824</InventTransId><ItemId>0033791</ItemId><LineAmount>1836.00</LineAmount><LineNum>1.0000000000</LineNum><Name>n-Hexan 95+ Pestapur, 2,5 l</Name><OrigPurchId>N20-03974</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP20-03595</PurchaseOrderId><purchId>N20-03974</purchId><PurchOrderDate>2020-03-17</PurchOrderDate><PurchPrice>459.00</PurchPrice><PurchUnit>ks</PurchUnit><Qty>4.000</Qty><RecId>5638507294</RecId><RecVersion>1193867781</RecVersion><TaxAmount>385.56</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03993824</InventTransId><Qty>4.00</Qty><RecId>5641764971</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP20-03595</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks></VendPurchOrderJour></PurchaseRequisition></MessageParts></Body></Envelope>