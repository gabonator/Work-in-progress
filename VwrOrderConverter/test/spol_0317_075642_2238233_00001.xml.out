BEGIN;N20-03974-1;Spolchemie;VWR;ORDERS;20200317;102511;HEAD;N20-03974-1;20200317;;CZK;10902SOLDTO;26000135;Spolek pro chemickou a hutní výrobu, akciová společnostSHIPTO;26005971;Spolek pro chemickou a hutní výrobu, akciová společnost;;;;Kekulova/Sklářská;;;;;Ústí nad Labem;400 32;CZ;korinkova@spolchemie.cz;477 164 632LINE;000001;;24575.320;n-Hexan 95+ Pestapur, 2,5 l;4.000;;20200317;459;1836.00END;N20-03974-1